typedoc.js

```diff
@@ -101,14 +75,157 @@ const entryPoints = packages
 
 const exclude =
   packages.flatMap(p => [`packages/${p}/test`]) +
-  ['packages/application-extension/src/index.tsx', 'examples/example.spec.ts'];
+  ['packages/application-extension'];
 
 module.exports = {
   entryPoints,
   exclude,
+  externalSymbolLinkMappings: {
```

Comment:
That's amazing! For anyone interested to see how it looks like:

![Screenshot from 2023-04-12 20-47-32](https://user-images.githubusercontent.com/5832902/231568603-f5cdd45d-337b-4b7b-a64d-43ab40430c37.png)